README.md

```diff
@@ -18,90 +18,90 @@ provide their own model implementations and use GraphStorm training pipeline to
-
-**Start the GraphStorm docker container**
-First, start your docker container by running the following command:
-
-```nvidia-docker run --network=host -v /dev/shm:/dev/shm/ -d --name test <graphstomr-image-name>```
-
-After running the container as a daemon, you need to connect to your container:
-
-```docker container exec -it test /bin/bash```
+**Note**: we assume users have setup a GraphStorm standalone environment following the [Setup GraphStorm with pip Packages](https://graphstorm.readthedocs.io/en/latest/install/env-setup.html#setup-graphstorm-with-pip-packages) instructions. And users git clone the GraphStorm source code into the `/graphstorm/` folder to use some complimentatry tools.
 
 **Node classification on OGB arxiv graph**
 First, use the below command to download the [OGB arxiv](https://ogb.stanford.edu/docs/nodeprop/#ogbn-arxiv) data and process it into a DGL graph for the node classification task.
 
-```python3 /graphstorm/tools/gen_ogb_dataset.py --savepath /tmp/ogbn-arxiv-nc/ --retain-original-features true```
+```
+python /graphstorm/tools/gen_ogb_dataset.py --savepath /tmp/ogbn-arxiv-nc/ --retain-original-features true
+```
 
 Second, use the below command to partition this arxiv graph into a distributed graph that GraphStorm can use as its input.
 
 ```
-python3 /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
-                                             --filepath /tmp/ogbn-arxiv-nc/ \
-                                             --num-parts 1 \
-                                             --num-trainers-per-machine 4 \
-                                             --output /tmp/ogbn_arxiv_nc_train_val_1p_4t
+python /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
+                                            --filepath /tmp/ogbn-arxiv-nc/ \
+                                            --num-parts 1 \
+                                            --num-trainers-per-machine 4 \
+                                            --output /tmp/ogbn_arxiv_nc_train_val_1p_4t
 ```
 
-GraphStorm distributed training relies on ssh to launch training jobs. These containers run ssh services in port 2222. Users need to collect the IP addresses of all machines and put all IP addresses in an ip_list.txt file, in which every row is an IP address. We suggest users to provide the ip_list.txt file’s absolute path in the launch script. If run GraphStorm training in a single machine, the ip_list.txt only contains one row as below.
+GraphStorm training relies on ssh to launch training jobs. The GraphStorm standalone mode uses ssh services in port 22. 
+
+In addition, users need to collect the IP addresses of all machines and put all IP addresses in an ip_list.txt file, in which every row is an IP address. We suggest users to provide the ip_list.txt file’s absolute path in the launch script. If run GraphStorm training in a single machine, the ip_list.txt only contains one row as below.
```

Comment:
If we are using standalone mode, does a user need to collect IP addresses of all machines?

Reply:
Removed the IP address collection sentence. Just leave the ip_list.txt file description.